eclipse/sw360
# feat(OAuthClient): Create, update, delete OAuthClient from UI

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*#1135*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: closes #1135 

### How To Test?
> - Import `Private_Pages_7_3_3_GA4.lar` file
> - Test `OAuth Client` Page under Admin Section.
> - Only admin should be able to access the page
> - Create, Edit, Delete OAuth Client.
> - Test token generation
![Capture2](https://user-images.githubusercontent.com/56534710/111117407-7c772c80-858d-11eb-80f8-e26fa0dcb4ee.JPG)
![Capture](https://user-images.githubusercontent.com/56534710/111117414-7da85980-858d-11eb-8d62-089195290a5a.JPG)

> Have you implemented any additional tests? - No

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
